cookbooks/bcpc/templates/default/network-interface-pre-up.sh.erb

```diff
@@ -16,6 +16,18 @@ if ! grep 4 $MODE_PATH; then
     echo 4 > $MODE_PATH
 fi
 
+LACP_RATE_PATH=/sys/class/net/<%= @interface %>/bonding/lacp_rate
+if ! grep 1 $LACP_RATE_PATH; then
```

Comment:
Yep. Those greps should be silent. Fixed that and added quoting around existing + new strings.